src/consent-manager/dialog.tsx

```diff
@@ -148,12 +160,12 @@ const Dialog: FC<Props> = ({ onCancel = undefined, onSubmit, title, innerRef, ch
     innerRef(container)
 
     return () => {
-        document.body.removeEventListener('keydown', handleEsc, false)
-        document.body.style.overflow = ''
-        document.body.removeChild(container)
-        innerRef(null)
+      document.body.removeEventListener('keydown', handleEsc, false)
+      document.body.style.overflow = ''
+      container && document.body.removeChild(container)
```

Comment:
don't you want also do a setContainer(undefined) ?

Reply:
added